### ISSUE TITLE
Поддержка клавиатур для ботов

### ISSUE THREAD

**Issue by Plasticable** [VKCOM/vk-java-sdk#116]
В библиотеке планируется поддержка [клавиатур для ботов](https://vk.com/dev/bots_docs_3)?

**Comment by rozhok** [comment]
Судя по всему, проект не поддерживают. Если не хотите пользоваться другими форками, где добавили поддержку клавиатур, то можете передавать их самостоятельно используя unsafeParam для сообщений.

Например:

```
String keyboard = "{\n" +
        "  \"one_time\": false,\n" +
        "  \"buttons\": [\n" +
        "    [\n" +
        "      {\n" +
        "        \"action\": {\n" +
        "          \"type\": \"text\",\n" +
        "          \"payload\": \"{\\\"button\\\": \\\"1\\\"}\",\n" +
        "          \"label\": \"Red\"\n" +
        "        },\n" +
        "        \"color\": \"negative\"\n" +
        "      },\n" +
        "      {\n" +
        "        \"action\": {\n" +
        "          \"type\": \"text\",\n" +
        "          \"payload\": \"{\\\"button\\\": \\\"2\\\"}\",\n" +
        "          \"label\": \"Green\"\n" +
        "        },\n" +
        "        \"color\": \"positive\"\n" +
        "      }\n" +
        "    ]\n" +
        "  ]\n" +
        "} ";

vkApi.messages()
        .send(vkGroup, userId)
        .message("Есть клавиатура!")
        .unsafeParam("keyboard", keyboard)
        .execute();
```

**Comment by xISRAPILx** [comment]
@rozhok, если с передачей клавиатуры всё просто, то с получением payload'а из ответа уже появляются проблемы.

**Comment by rozhok** [comment]
Да, payload не приходит, но дело ведь не в SDK, так как это обычный HTTP запрос на вебхук бота. А при лонг-полле payload тоже не приходит?

**Comment by rozhok** [comment]
@xISRAPILx чтобы получить payload нужно поднять версию API до 5.69 или выше (https://vk.com/dev/bots_docs_3)

**Comment by Pushkin23** [comment]
> 
> 
> @xISRAPILx чтобы получить payload нужно поднять версию API до 5.69 или выше (https://vk.com/dev/bots_docs_3). Проверил — работает (что логично :) )

Все таки не подскажите, как получить payload?

**Comment by rozhok** [comment]
В смысле - как? Как только вы подымете версию апи в настройках, на ваш
вебхук начнут приходить запросы с payload.

**Comment by minecraft8997** [comment]
Отправил клавиатуру вот так: https://github.com/VKCOM/vk-java-sdk/issues/116#issuecomment-470300090. Выдало ошибку This is a chat bot feature, change this status in settings. Использую Long Poll Api. Что делать?

**Comment by rozhok** [comment]
@minecraft8997 в настройках сообщества, в сообщениях:
![image](https://user-images.githubusercontent.com/1721631/57560741-8677e500-7390-11e9-857f-475a38ae401b.png)
![image](https://user-images.githubusercontent.com/1721631/57560747-8e378980-7390-11e9-8ae0-06e313cdcc02.png)
